jvm/src/test/kotlin/app/cash/trifle/CertificateTests.kt

```diff
@@ -0,0 +1,52 @@
+package app.cash.trifle
+
+import app.cash.trifle.testing.TestCertificateAuthority
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Assertions.*
+import java.security.*
+
+internal class CertificateTests {
+  @Nested
+  @DisplayName("Certificate#verify() Tests")
+  inner class CertificateVerifyTests {
+    @Test
+    fun `test verify() returns true for a properly issued certificate`() {
+      assertTrue(
+        endEntity.certificate.verify(
+          endEntity.certRequest,
+          endEntity.certChain.drop(1),
```

Comment:
with the change ^ dont have to explicitly do a `.drop(1)` which is actually dropping the end entity certificate rather than the attempt to drop the anchor (indexed 1)

Reply:
Again, this is deliberate per offline discussion.